### PR TITLE
Set up Percy

### PR DESCRIPTION
**Pull request by sree-cfa**
Set up [percy](https://percy.io/Code-for-America/snlab-il-ccap)

### CODE REVIEW

File: src/test/java/org/ilgcc/app/utils/Page.java [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+
+import io.percy.selenium.Percy;
```
Comment (coltborg) [inline src/test/java/org/ilgcc/app/utils/Page.java:9]: Where does this package come from? I don't see it defined in `build.gradle`, is it coming from the JavaScript dev dependency?

Comment (coltborg) [reply src/test/java/org/ilgcc/app/utils/Page.java:9]: Hmm we'll they're showing up in the [Percy project](https://percy.io/Code-for-America/snlab-il-ccap/), that's a good sign.

Comment (sree-cfa) [reply src/test/java/org/ilgcc/app/utils/Page.java:9]: It's here https://github.com/codeforamerica/il-gcc-form-flow/blob/main/build.gradle#L75

Comment (coltborg) [reply src/test/java/org/ilgcc/app/utils/Page.java:9]: 🤦🏻‍♀️ Ty, I was looking at the library gradle file.

**Review by coltborg — APPROVED** [review]
I can see percy adding screenshots in https://percy.io/Code-for-America/snlab-il-ccap

**Pull request merged**